sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TelemetryPartATests.cs

```diff
@@ -110,7 +112,14 @@ public void InitRoleInfo_ServiceNameAndInstanceAndNamespace()
         [Fact]
         public void GeneratePartAEnvelope_DefaultActivity_DefaultResource()
         {
-            var activity = CreateTestActivity();
+            using ActivitySource activitySource = new ActivitySource(ActivitySourceName);
+            var activity = activitySource.StartActivity(
+                ActivityName,
+                ActivityKind.Client,
+                parentContext: default,
+                startTime: DateTime.UtcNow);
+            activity.Stop();
```

Comment:
not required? as the rest of the test doesn't depend on whether the activity is stopped or not.